sources/packages/web/src/components/common/students/StudentAccountRestrictions.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <body-header-container :enableCardView="true">
+    <template #header>
+      <body-header title="Your account" />
+    </template>
+    <template v-if="restrictions.length > 0">
+      <p>
+        <span class="font-bold">Please resolve the items below soon</span>
+        to minimize the disruption and impact.
+      </p>
+      <content-group
+        ><ul v-for="(restriction, index) in restrictions" :key="index">
+          <li>{{ restriction.code }}</li>
+        </ul>
+      </content-group>
+    </template>
+    <template v-else><p>There are no items to resolve.</p></template>
+  </body-header-container>
+</template>
+<script lang="ts">
+import { computed, onMounted, defineComponent } from "vue";
+import { StudentRestriction } from "@/store/modules/student/student";
+import { useStudentStore } from "@/composables";
+
+interface StudentAccountActivityFormModel {
+  restrictions: StudentRestriction[];
+}
+
+export default defineComponent({
+  setup() {
+    const { activeRestrictions, updateRestrictions } = useStudentStore();
+    const restrictions = computed<StudentAccountActivityFormModel>(
```

Comment:
The computed output id of type `StudentRestriction[]` and not `StudentAccountActivityFormModel`

![image](https://github.com/user-attachments/assets/834add6a-c621-44f0-b14c-b42f52da66c0)

After: 

![image](https://github.com/user-attachments/assets/a188175c-5710-4f31-9293-ee9576632903)

and the interface `StudentAccountActivityFormModel` can be removed.